### PR TITLE
Sveltekit Upgrade

### LINKED ISSUE #3208: Upgrade to Sveltekit `359`
Refactor `svelte.config.js` as the vite-config file gets intrduced

[**_Official Changelog_**](https://github.com/sveltejs/kit/blob/master/packages/kit/CHANGELOG.md#100-next359)

### PR DESCRIPTION
**Pull request by Cahllagerfeld**
closes #3208

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3277"></a>

### CODE REVIEW

**Review by Cahllagerfeld — COMMENTED** [review]
From time to time I see in the console:
```sh
Error writing stream Error [ERR_STREAM_DESTROYED]: Cannot call write after a stream was destroyed
    at new NodeError (node:internal/errors:387:5)
    at write_ (node:_http_outgoing:851:11)
    at ServerResponse.write (node:_http_outgoing:808:15)
    at next (file:///workspace/website/node_modules/@sveltejs/kit/dist/node.js:5839:8) {
  code: 'ERR_STREAM_DESTROYED'
}
```

I can imageine it's caused by [Version 356](https://github.com/sveltejs/kit/blob/master/packages/kit/CHANGELOG.md#100-next356). I can't say how bad this is, or if it's just affecting the dev-server.

**Review by Cahllagerfeld — COMMENTED** [review]
In the Netlify Logs whenever an endpoint gets called we can see the log.
That's not an issue in my opinion, but as it's marked as an error, the netlify log trigger
```sh
 ERROR  (node:8) ExperimentalWarning: buffer.Blob is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
```

When running on the dev-server that's not the case

File: package.json [hunk 1]
```diff
@@ -49,7 +48,7 @@
     "@netlify/functions": "^1.0.0",
     "@octokit/plugin-paginate-graphql": "^2.0.1",
     "@sveltejs/adapter-netlify": "1.0.0-next.69",
```
Comment (Siddhant-K-code) [inline package.json:50]: Do you think we should bump [this](https://github.com/sveltejs/kit/blob/master/packages/adapter-netlify/CHANGELOG.md#100-next79) up also?

**Comment by Siddhant-K-code** [comment]
> In the Netlify Logs whenever an endpoint gets called we can see the log.
> That's not an issue in my opinion, but as it's marked as an error, the netlify log trigger
> ```sh
>  ERROR  (node:8) ExperimentalWarning: buffer.Blob is an experimental feature. This feature could change at any time
> (Use `node --trace-warnings ...` to show where the warning was created)
> ```
> 
> When running on the dev-server that's not the case

I just googled it. Seems to have a know issue with svelte & Netlify. See [this](https://answers.netlify.com/t/debugging-advice-when-trying-to-deploy-sveltekit-to-netlify-cannot-find-module-llhttp-llhttp-simd-wasm-js-from-opt-build-repo-netlify/70397) 👀 

Looks like No Action Item for us, except this [comment suggestion](https://github.com/sveltejs/kit/issues/5337#issuecomment-1175702558)

File: package.json (hunk 1 above)
Comment (Cahllagerfeld) [reply package.json:50]: The next adapter update comes with kit version `380` I think

**Comment by Cahllagerfeld** [comment]
I'm a bit on the fence, because we used the same version already without issues, so I assume it's not the adapter version 🤔

**Comment by Cahllagerfeld** [comment]
> From time to time I see in the console:
> 
> ```shell
> Error writing stream Error [ERR_STREAM_DESTROYED]: Cannot call write after a stream was destroyed
>     at new NodeError (node:internal/errors:387:5)
>     at write_ (node:_http_outgoing:851:11)
>     at ServerResponse.write (node:_http_outgoing:808:15)
>     at next (file:///workspace/website/node_modules/@sveltejs/kit/dist/node.js:5839:8) {
>   code: 'ERR_STREAM_DESTROYED'
> }
> ```
> 
> I can imageine it's caused by [Version 356](https://github.com/sveltejs/kit/blob/master/packages/kit/CHANGELOG.md#100-next356). I can't say how bad this is, or if it's just affecting the dev-server. I can't reproduce it consistently currently

I was able to reproduce it on another Sveltekit-Project with the same version, so I assume it's not an issue inside the project 🤔

**Review by Cahllagerfeld — COMMENTED** [review]
Remix seems to have the same problem with the logs [1](https://github.com/remix-run/remix/issues/1141) It seems to be nothing to worry about. It's still a bit annoying though

**Pull request merged**